### PR TITLE
Rc profile info

### PR DESCRIPTION
**Pull request by robert-czwartosz**
I created component with profile information.
To test it run: npm run start:profileInfo

### CODE REVIEW

File: Src/UIComponents/ProfileInfo/ProfileInfo.ts [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+    return element;
+}
+
+function profileInfo(profileData: {
+    username: string,
+    gender: 'Male' | 'Female',
+    dateOfBirth: Date,
+    height: number, // cm
+    weight: number, // kg
+    activityLevel: 'Low' | 'Regular' | 'Active' | 'Very Active',
```
Comment (skorpiom) [inline Src/UIComponents/ProfileInfo/ProfileInfo.ts:18]: Hey, here is an enum, you could use to indicate options:
https://github.com/KrystianKjjk/CodersCamp2020.Project.TypeScript.FitNotFat/blob/IP_Models/Models/ActivityLevel.model.ts

**Comment by skorpiom** [comment]
Looks great, I left just one small suggestion.

**Review by fszwedo — APPROVED** [review]
all good from my side

File: Src/UIComponents/ProfileInfo/ProfileInfo.ts (hunk 1 above)
Comment (robert-czwartosz) [reply Src/UIComponents/ProfileInfo/ProfileInfo.ts]: Good point. Your suggestion is applied.
If you don't have any additional suggestions, you may merge this branch to master.

**Pull request merged**